frontend/src/components/pages/FilterBox.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Grid, GridItem, Stack } from "@chakra-ui/react";
+import React, { useState } from "react";
+
+import { Option } from "../../types/BookTypes";
+import AddMultiSelect from "./CreateReview/AddMultiSelect";
+
+interface FilterBoxProps {
+  genreOptions: Option[]; // dropdown of genres to choose from
+  ageOptions: Option[]; // dropdown of ages to choose from
+  setGenreFilter: (genreFilter: Option[]) => void;
+  setAgeFilter: (ageFilter: Option[]) => void;
+}
+
+const FilterBox = ({
+  genreOptions,
+  ageOptions,
+  setGenreFilter,
+  setAgeFilter,
+}: FilterBoxProps): React.ReactElement => {
+  const [genres, setGenres] = useState<Option[]>([]); // save the genres choosen
+  const [age, setAge] = useState<Option[]>([]); // save the ages choosen
+
+  const clickApply = async () => {
+    setGenreFilter(genres);
+    setAgeFilter(age);
+  };
+
+  const clear = async () => {
+    setGenres([]);
+    setAge([]);
+    setGenreFilter([]);
+    setAgeFilter([]);
+  };
+
+  return (
+    <Grid templateColumns="repeat(2, 1fr)" gap={4} borderColor="110A23">
+      <GridItem>
+        <Stack direction="row" width="100%" spacing={4}>
+          <AddMultiSelect
+            id="Audience"
+            label=""
+            placeholder="Audience"
+            options={ageOptions}
+            setOptions={null}
+            optionsSelected={age}
+            setOptionsSelected={setAge}
+            allowMultiSelectOption={false}
+            allowAddOption={false}
+          />
+          <AddMultiSelect
+            id="genre"
+            label=""
+            placeholder="Genres"
+            options={genreOptions}
+            setOptions={null}
+            optionsSelected={genres}
+            setOptionsSelected={setGenres}
+            allowMultiSelectOption
+            allowAddOption={false}
+          />
+          <Button onClick={clickApply}>Apply</Button>
```

Comment:
I think these two "Apply" and "Clear" can be centered vertically of the boxes: they're a little more towards the top right now.
<img width="485" alt="Screen Shot 2022-08-03 at 7 17 03 PM" src="https://user-images.githubusercontent.com/53016529/182728557-147df27b-191e-41c6-9812-63e49a698bf0.png">

Reply:
Do we need to make this responsive for mobile view? Looks a bit squished right now
<img width="289" alt="Screen Shot 2022-08-03 at 7 23 32 PM" src="https://user-images.githubusercontent.com/53016529/182729189-67a05ef8-b4d5-46e4-948a-2891d2ab4162.png">
.

Reply:
I think @susheelk mentioned that the design of the filter boxes and buttons is not included in this ticket 🤔 I could work on it tho if needed

Reply:
I think it's a good idea to make this somewhat responsive, even if it doesn't have to look good. Maybe have the buttons flow to a new line?